src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserResource.kt

```diff
@@ -32,12 +39,87 @@ class UserResource(
   private val userService: UserService,
   private val authenticationFacade: AuthenticationFacade,
 ) {
+  @PreAuthorize("hasRole('ROLE_CREATE_USER')")
+  @PostMapping("")
+  @ResponseStatus(HttpStatus.CREATED)
+  @Operation(
+    summary = "Create user",
+    description = "Create user",
+    security = [SecurityRequirement(name = "CREATE_USER")],
+    requestBody = io.swagger.v3.oas.annotations.parameters.RequestBody(
+      content = [
+        Content(
+          mediaType = "application/json",
+          schema = Schema(implementation = CreateUserRequest::class)
+        )
+      ]
+    ),
+    responses = [
+      ApiResponse(
+        responseCode = "201",
+        description = "User Information Returned",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = UserDetail::class))]
```

Comment:
I think I found you don't need to explicitly add the `content` attribute since it can work it out

Reply:
Ok